src/components/trends/common/Tooltip.css

```diff
@@ -0,0 +1,40 @@
+.tooltip {
+  display: block;
+  font-size: 14px;
+  background-color: #FFFFFF;
+  border: solid;
+  position: absolute;
+  border-radius: 4px;
+  opacity: 0.75;
+}
+
+.content {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 5px 10px;
+  box-sizing: border-box;
+}
+
+.content span {
+  width: 100%;
+}
+
+.title {
+  composes: mediumContrastText from '../../../styles/typography.css';
+  composes: content;
+  background-color: var(--tooltip-title-bg);
+}
+
+.tail {
+  width: 0px;
+  height: 0px;
+  border-style: solid;
+  border-color: transparent;
```

Comment:
OK, so as a check of my understanding...the `transparent` here is to cover all sides not overridden with a color in the JavaScript via higher-priority inline styles?